### ISSUE TITLE
[Server]: Renamed Adapter not used for Hosting

### ISSUE THREAD

**Issue by Crashdummyy** [DoM1niC/PlayHide#18]
After Creating a new Adapter with
`tapinstall.exe install "C:\Program Files\TAP-Windows\driver\OemVista.inf" tap0901`

and Renaming it to PlayHide Server.
Hosting Server still takes the PlayHide VPN Adapter.

The Setting was altered that way:
[Server]
AdapterName="PlayHide Server"

Naming it PlayHideServer wont help either

**Comment by DoM1niC** [comment]
Do you have one or a second Adapter ?

**Comment by Crashdummyy** [comment]
Ich habe einen 2. Adapter über tabinstall.exe install erstellt.
Den habe ich über die WIndowsFunktion und danach auch über Powershell umbenannt.
Name: PlayHide Server
IntefaceDescription: Tap Windows Adapter #2

**Comment by DoM1niC** [comment]
@Crashdummyy Yep das Umbenennen ist derzeit auf ein Adapter festgelegt, Optimierungen für Multiadapter muss ich nochmal dazu rüsten, wenn du weitere diverse Fragen oder Anregungen hast, schreib mir einfach über meinen Livechat https://chat.3dns.eu/livechat?version=1.0.0&mode=popout

**Comment by DoM1niC** [comment]
I included the Server Stuff into the Client in Config this could be configured like the Server Version but this no more needed, the Interface Name will be PlayHide VPN normal from the Appname currently, more customizations will follow...
